sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonsView.java

```diff
@@ -277,4 +290,54 @@ public HorizontalLayout createAssociationFilterBar() {
 
 		return associationFilterLayout;
 	}
+
+	private PersonAssociation getFirstAllowedPersonAssociation() {
+
+		Iterator<String> associationsIterator = associationButtons.values().iterator();
+		PersonAssociation defaultAssociation = associationsIterator.hasNext() ? PersonAssociation.valueOf(associationsIterator.next()) : null;
+		if (defaultAssociation == PersonAssociation.ALL) {
+			defaultAssociation = associationsIterator.hasNext() ? PersonAssociation.valueOf(associationsIterator.next()) : null;
+		}
+		return defaultAssociation;
+	}
+
+	private boolean isPersonAssociationAllowed(PersonAssociation personAssociation) {
```

Comment:
The logic looks correct, but I think the calculation should be moved to a separate method (will be replaced by a following ticket).

Reply:
Does not need to be done because this logic will be moved to the backend anyway in #10414